src/components/VersionRestartModal/TaskStatusCheckbox.tsx

```diff
@@ -28,11 +27,9 @@ const CheckboxComponent: React.FC<TaskStatusCheckboxProps> = ({
     name={taskId}
     label={
       <StateItemWrapper>
-        <PaddedSquare color={mapTaskStatusToColor[status]} />
-        {baseStatus && (
-          <PaddedSquare color={mapTaskStatusToColor[baseStatus]} />
-        )}
-        {displayName}
+        <StyledTaskStatusIcon status={status} />
+        {baseStatus && <StyledTaskStatusIcon status={baseStatus} />}
+        <div>{displayName}</div>
```

Comment:
If you wrap <StyledTaskStatusIcon /> with a fixed width wrapper it can handle positioning of the icons and prevent the `displayName` text from being pushed to the side when `baseStatus` doesn't exist for a task. 

So we can avoid situations where the list is not aligned.